cpan/Digest-SHA/lib/Digest/SHA.pm

```diff
@@ -11,8 +11,10 @@ use integer;
 $VERSION = '5.98';
 
 require Exporter;
-require DynaLoader;
-@ISA = qw(Exporter DynaLoader);
+
+use XSLoader ();
+
+@ISA = qw(Exporter);
```

Comment:
Shouldn't changes like this be submitted first on CPAN?

Reply:
Yes but the point of this PR was to show what we did to the tarball during build.